10LessonTen/Checkpoint2/css/style.css

```diff
@@ -0,0 +1,209 @@
+main {
+    background-color: background-color: rgba(251, 208, 192, 0.86);
+}
+nav {
+    background-color: rgba(251, 208, 192, 0.86);
+    padding: 20px;
+    width: 100%;
+}
+.opacity {
+    background-color: rgba(0, 0, 0, 0.3);
+    width: 100%;
+    color: white;
+    height: 90vh;
+}
```

Comment:
I can side scroll when i view your site which means there's elements that add up to more than 100% width!